debian/rules

```diff
@@ -58,18 +58,19 @@ override_dh_auto_configure:
 		$(CONF_LUA) \
 		$(CONF_PIM6) \
 		$(CONF_GRPC) \
+		--with-protobuf \
 		--with-libpam \
 		--enable-doc \
 		--enable-doc-html \
 		--enable-snmp \
 		--enable-fpm \
-		--disable-protobuf \
 		--disable-zeromq \
 		--enable-ospfapi \
 		--enable-bgp-vnc \
 		--enable-multipath=256 \
 		--enable-pcre2posix \
-		\
+		--enable-mgmtd-test-be-client \
+		--enable-fpm-listener \
```

Comment:
fpm-listener probably should be it's own commit or spelled out the reason for the inclusion here.  Since it is not related to protobuf changes at all.